### ISSUE TITLE
iOS mobile: Keyboard doesn't appear/can't be called up

### ISSUE THREAD

**Issue by MediaSmarts** [DoYourThing/CyberChoices#249]

**Comment by robgungor** [comment]
Yes you can't just pull up a keyboard any any website, it's only for text entry.

**Comment by MediaSmarts** [comment]
So how can users use the VI interface or switch between English and French?

**Comment by robgungor** [comment]
Not sure... I think this was a design oversight.

For the French and English maybe there should be an additional prompt... For the VI mode I think we can assume they can't use on mobile- or will have to use built in phone accessibility features to navigate.

**Comment by MediaSmarts** [comment]
Hm -- I'll have to look into that. Thanks.

**Comment by MediaSmarts** [comment]
Hi Rob, we spoke to an accessibility expert and got this response:

"As your game "talks" I"d add a vocal command "next" "previous" for people with hearing problems. Annyang supports multiple languages, and as your game is online, the voice recognition service will be always available too.
 
Or I'd add a "touchmove" listener and when the finger moves over the button I'd generate a click or a voice that says click to go next... 
 
https://github.com/TalAter/annyang "

Could you look into that please? (I realize this probably counts as an enhancement rather than a bug, but it would be helpful to get an estimate from you on how much time and hassle would be involved.)

**Comment by robgungor** [comment]
Seems pretty cool - definitely is an enhancement, though and might take a bit to define and implement all commands - especially with 2 language's going on.

**Comment by MediaSmarts** [comment]
OK, leave it for now -- for the moment we want to focus on dealing with the remaining issues in Chrome and Firefox that are keeping us from doing classroom testing.
